CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
+- Fixed Wallet panel from being closed also when user dismisses the pending quest panel. ([#7505](https://github.com/brave/brave-ios/issues/7505))
+
+### General  
+
+- Implemented bookmark count feature for bookmark folders. ([#7674](https://github.com/brave/brave-ios/issues/7674))
+- Added support for using custom search engine in Private Mode. ([#7733](https://github.com/brave/brave-ios/issues/7733))
+- Added “Media” settings menu in main settings. [#7677](https://github.com/brave/brave-ios/issues/7677)
+- Added “Highest Quality Playback” preference in “Media” setting for YouTube video resolution. ([#7055](https://github.com/brave/brave-ios/issues/7055))
+- Added voice search feature in the URL bar. ([#7032](https://github.com/brave/brave-ios/issues/7032))
+- Added new search engines for South Korea region. ([#6912](https://github.com/brave/brave-ios/issues/6912))
+- Added support for using component updater to fetch background and sponsored images. ([#7467](https://github.com/brave/brave-ios/issues/7467))
+- Added Brave Talk Jitsi SDK Integration to release channel builds. ([#7560](https://github.com/brave/brave-ios/issues/7560))
+- Replaced Playlist onboarding tutorial with the new revamped version. ([#7526](https://github.com/brave/brave-ios/issues/7526))
+- Updated the certificate viewer sizing to properly display some fields which were being cropped out. ([#7603](https://github.com/brave/brave-ios/issues/7603))
+- Updated Youtube setting strings in “Media” settings menu. ([#7776](https://github.com/brave/brave-ios/issues/7776))
+- Updated status bar overlay color to match the Brave UI in reader mode. ([#7673](https://github.com/brave/brave-ios/issues/7673))
+- Updated URL bar context menus to display operations in reverse order for “Bottom Bar” mode. ([#6260](https://github.com/brave/brave-ios/issues/6260))
+- Updated the order of buttons in the “Start a sync chain” flow. ([#4821](https://github.com/brave/brave-ios/issues/4821))
+- Resolved audio files not being added to Playlist using long press gesture. ([#7667](https://github.com/brave/brave-ios/issues/7667))
+- Resolved ad notifications not working as expected. ([#7586](https://github.com/brave/brave-ios/issues/7586))
+- Resolved displaying correct settings screen after unlocking browser. ([#7657](https://github.com/brave/brave-ios/issues/7657))
+- Resolved security issue of OpenSearch allowing to add rogue suggestion providers as reported on HackerOne by nishimunea. ([#7702](https://github.com/brave/brave-ios/issues/7702))
+- Fixed script blocking embedded video player on https://puredarwin.org. ([#7822](https://github.com/brave/brave-ios/issues/7822))
+- Fixed crash when launching browser with hundreds of tabs open. ([#7804](https://github.com/brave/brave-ios/issues/7804))
+- Fixed voice search popup presentation on iPad when “Split View” is enabled. ([#7693](https://github.com/brave/brave-ios/issues/7693))
```

Comment:
Probably should use onboarding rather than presentation. For example:

```
- Fixed voice search onboarding on iPad when “Split View” is enabled. ([#7693](https://github.com/brave/brave-ios/issues/7693))
```